linter.py

```diff
@@ -35,7 +35,7 @@ class ESLint(NodeLinter):
     )
     line_col_base = (1, 1)
     defaults = {
-        'selector': 'source.js - meta.attribute-with-value, text.html.basic'
+        'selector': 'source.js - meta.attribute-with-value'
```

Comment:
What the hell is this meta thing. I don't need that, do I?

Reply:
javascript attributes like `onchange` are also scoped as source.json, but it doesn't make sense to lint them because e.g. of course the function isn't defined in the same file.

<img width="454" alt="screen shot 2018-03-03 at 20 48 47" src="https://user-images.githubusercontent.com/2543659/36938611-71f14822-1f24-11e8-9e42-baa90464a353.png">

Reply:
That selector stuff is relatively powerful

Reply:
It's probably too powerful 😁